tests/tian_peridotite.prm

```diff
@@ -0,0 +1,125 @@
+# This model generates a representation of a phase diagram for the equilibrium
+# wt% water that can be stored in an upper mantle peridotite (PUM) as shown in
+# Figure 11 of  Tian et. al., 2019. This model reproduces that phase diagram
+# (very low res by default), with an important caveat being that the pressure axis
+# is inverted relative to the ASPECT output. A surface pressure of 600 MPa is imposed
+# to reproduce the pressure axis,and a lateral temperature gradient is imposed to
+# reproduce the temperature axis.
+set Nonlinear solver scheme                = iterated Advection and Stokes
+set Max nonlinear iterations               = 10
+set Nonlinear solver tolerance             = 1
```

Comment:
Are you sure this is what you want?

Reply:
Definitely not!! Thanks for catching that